docs/components/badges.html

```diff
@@ -0,0 +1,164 @@
+                    <span>Favorite Question</span>
+                </span>
+                <span class="grid--cell s-badge">
+                    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeBronze.svg" aria-hidden="true"></span>
+                    <span>Altruist</span>
+                </span>
+            </div>
+        </div>
+    </div>
+
+    <h3 class="grid jc-space-between ai-center stacks-h3"><div class="grid--cell fl1">Tag Badge</div></h3>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<span class="s-badge s-badge__tag">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeGold.svg" aria-hidden="true"></span>
+    <span>css</span>
+</span>
+<span class="s-badge s-badge__tag">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeSilver.svg" aria-hidden="true"></span>
+    <span>javascript</span>
+</span>
+<span class="s-badge s-badge__tag">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeBronze.svg" aria-hidden="true"></span>
+    <span>swift</span>
+</span>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <div class="grid gs4">
+                <span class="grid--cell s-badge s-badge__tag">
+                    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeGold.svg" aria-hidden="true"></span>
+                    <span>css</span>
+                </span>
+                <span class="grid--cell s-badge s-badge__tag">
+                    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeSilver.svg" aria-hidden="true"></span>
+                    <span>javascript</span>
+                </span>
+                <span class="grid--cell s-badge s-badge__tag">
+                    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeBronze.svg" aria-hidden="true"></span>
+                    <span>swift</span>
+                </span>
+            </div>
+        </div>
+    </div>
+
+    <h3 class="grid jc-space-between ai-center stacks-h3"><div class="grid--cell fl1">Badge Counts</div></h3>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<span class="s-badge s-badge__gold">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeGold.svg" aria-hidden="true"></span>
+    <span>635</span>
+</span>
+<span class="s-badge s-badge__silver">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeSilver.svg" aria-hidden="true"></span>
+    <span>7624</span>
+</span>
+<span class="s-badge s-badge__bronze">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeBronze.svg" aria-hidden="true"></span>
+    <span>8234</span>
+</span>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <div class="grid gs4">
+                <span class="grid--cell s-badge s-badge__gold">
+                    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeGold.svg" aria-hidden="true"></span>
```

Comment:
Can the class be applied directly to the `img` to reduce the markup?

Reply:
Sure